src/Changelog/Formatter/MarkdownFormatter.php

```diff
@@ -40,6 +41,10 @@ public function format(Changelog $changelog): string
             $lines[] = '';
 
             foreach ($changelog->getChangesForVersion($version) as $change) {
+                if (strpos($change, '[MERGE]') !== false) {
```

Comment:
Why is it filtered here AND in src/Changelog/PullRequestChangelogGenerator.php:82 ?

If it is already filtered in PullRequestChangelogGenerator this if should never return true right?

Reply:
True, this can be removed